contrail_config_functions

```diff
@@ -435,7 +435,7 @@ function replace_contrail_dns_conf()
 function check_dev()
 {
    interface=$1
-   arr=$(ifconfig | awk '{print $1}')
```

Comment:
This is true. The install Guides should add the export PATH=${PATH}:/sbin as a recommendation for the user's environment (typically "stack")